src/dropdown/dropdown.ts

```diff
@@ -102,9 +108,17 @@ export class NgbDropdown {
     this._zoneSubscription = ngZone.onStable.subscribe(() => { this._positionMenu(); });
   }
 
-  isUp() { return this.placement.indexOf('top') !== -1; }
+  isUp() {
```

Comment:
Maybe we should simplify those methods in the following way:
* `isDown()` is true only when there is `bottom` somewhere in placement
* `isUp()` = `!isDown()`

WDYT?

In any case we should tests for this logic.